features/filesystem/sd/SDBlockDevice.h

```diff
@@ -127,7 +127,7 @@ class SDBlockDevice : public BlockDevice {
      */
     virtual void debug(bool dbg);
 
-private:
+protected:
```

Comment:
I'd like to change  `_transfer_sck` to about 20 MHz to access fast to SDCard by inheriting [SDBlockDevice](https://github.com/ARMmbed/sd-driver/tree/master/features/filesystem/sd).

But, in mbed-os(inherits [SDBlockDevice](https://github.com/ARMmbed/sd-driver/tree/master/features/filesystem/sd)), I cannot change SPI clock rate with no changes because the access restriction is **private:** . Therefore, I need to change the access restriction in manual.
In mbed classic(inherits [SDFilesystem](https://developer.mbed.org/teams/mbed/code/SDFileSystem/)), I can change the clock with no changes because  the access restriction is **protected:** .

I think that `SDFileSystem` is the base of `SDBlockDevice`,
but would you please let me teach if you know the reason changed the access restriction?